### ISSUE TITLE
INSTALLATION ON WEB SERVER MUTUALISED

### ISSUE THREAD

**Issue by super-viny** [claroline/Distribution#1400]
Hello,

I try to install "Claroline" on a shared hosting.
I downloaded the package from github web site ("https://github.com/claroline/Claroline"). I've already tried several ways, download with "curl command" or by myself and thereafter extrat the archive with "tar command".
I modified the file "claroline/app/config/parameters.yml" and indicate the right hostname, username, password, database name to my database server and when I launch the installation using the script "/claroline/web/install.php" the file "parameters.yml" is systematically rewritten with default parameters as "root", "localhost", ... 
This goes without saying that the installation fails.
Is it me that makes something wrong for proceed with installing or does exist a way to change this behavior?

Many thanks for your help in advance,
Alain

**Comment by stefk** [comment]
Hello,

While it's listed in the available methods, the web installer approach isn't the most tested/reliable one at this point. I would suggest you use [that one](https://github.com/claroline/Claroline#1-from-a-pre-built-archive). At least you'll get some usable error output, if any. Also, we're about to release a new version, so unless it's just for testing, you might want to wait a couple of days.

**Comment by super-viny** [comment]
Hello,

Many thanks for your answer Stefk.

I have chosen the web installer because even if I can connect my host shared via a connection ssh, the command "composer" is not available and I am not sure that I could install it.

Eventually, I used the following commands:

tar xzvf claroline-16.05.tar.gz

cd claroline-16.05

php scripts/configure.php

chmod -R 777 app/sessions
chmod -R 777 app/cache
chmod -R 777 app/logs
chmod -R 777 files
chmod -R 777 web/uploads

php app/console claroline:install

php app/console assets:install web --symlink
php app/console assetic:dump

php app/console claroline:update

php app/console claroline:user:create -a

And I got a version usable.

Hope that could help somebody else...

Alan
